marlowe-playground-client/src/Simulation/State.purs

```diff
@@ -238,17 +259,21 @@ updateContractInStateP text state = case parseContract text of
             holes = view L._holes lintResult
           (set _holes holes) state
   Left error -> (set _holes mempty) state
+  where
+  marloweState = case state.executionState of
+    Just s -> s.state
+    Nothing -> emptyState zero
```

Comment:
interesting, you have used `maybe` a lot but not here?

Reply:
Good point. It is because I did it first and hadn't thought of that function until later :)